docs/Users_Guide/appendixH.rst

```diff
@@ -0,0 +1,63 @@
+.. _appendixH:
+
+*****************************
+Appendix H Unstructured Grids
+*****************************
+
+Introduction
+============
+
+METv12.0.0+ includes limited support for using datasets on unstructured grids (UGRIDs) in the PointStat and GridStat tools. This support includes verifying UGRID forecasts against point observations (PointStat), and verifying UGRID forecasts regridded to a structured grid against gridded observations on a structured grid (GridStat). The implementation of UGRID support in METv12.0.0+ is based on the UGRID Conventions (https://ugrid-conventions.github.io/ugrid-conventions/). These conventions define various elements for common UGRID topologies. Thus far, development to support datasets on unstructured grids has been driven by the LFRIc (“elfrick”) NWP model, and the Model for Prediction Across Scales (MPAS) NWP model. However, the support for unstructured grids was implemented in such a way that additional unstructured grids can be utilized, provided the topology can be related to the UGRID conventions.
```

Comment:
LFRIc --> LFRic. 

Also, maybe we're not ready to show this much background but could note that the grid is represented by Atlas library data structures. This paper is a good reference for Atlas: https://doi.org/10.1016/j.cpc.2017.07.006. It has a useful description of (un)structured grid heirarchy, and it gives definitions of cell/node/edge (see fig. 6). I'm wondering if in the future these might be a better way for us to refer to them than the "UGRID conventions" approach. In particular see the definition for "node" vs how it is used in other naming conventions. But, maybe best to discuss going in this direction in a future iteration.

Reply:
@willmayfield I will change `LFRic`, thanks.

My opinion at this point is to consider how to describe the unstructured grid hierarchy for future updates, using what you've written here as a basis for the discussion that maybe we have at the UGRID planning meeting that is forthcoming. It's not clear to me how Atlas is used right now by MET (if at all?), or if it will be used by MET in the future (when we move to the proj library?), so I lean toward keeping it general for now and hope early users have enough domain knowledge to get around. Does that seem OK?

Reply:
OR, maybe we remove mention of "UGRID conventions", and instead remove the notion that it's based on UGRID conventions and just generically describe the capability as a way for a user to "describe the topology of their UGRID"?

Reply:
I agree with keeping it general for now!